src/test/java/seedu/ccacommander/model/ModelManagerTest.java

```diff
@@ -194,6 +196,33 @@ public void updateEvents() {
         assertFalse(modelManager.hasEvent(AURORA_BOREALIS));
     }
 
+    @Test
+    public void hasEnrolment_nullEnrolment_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> modelManager.hasEnrolment(null));
+    }
+
+    @Test
+    public void hasEnrolment_enrolmentNotInCcaCommander_returnsFalse() {
+        assertFalse(modelManager.hasEnrolment(ALICE_AURORA));
+    }
+
+    @Test
+    public void hasEnrolment_enrolmentInCcaCommander_returnsTrue() {
+        modelManager.createEnrolment(ALICE_AURORA);
+        assertTrue(modelManager.hasEnrolment(ALICE_AURORA));
+    }
```

Comment:
Just to confirm, these have nothing to do with the new function written right but are meant to serve as additional test cases?

Reply:
yep, you are correct!